projects/demo-playwright/tests/legacy/input-number/input-number.pw.spec.ts

```diff
@@ -22,7 +22,7 @@ test.describe('InputNumber', () => {
         });
 
         test('Infinite precision', async ({page}) => {
-            await tuiGoto(page, `${DemoRoute.InputNumber}/API?precision=Infinity`);
+            await tuiGoto(page, `${DemoRoute.InputNumberLegacy}/API?precision=Infinity`);
```

Comment:
All changes inside this file – `DemoRoute.InputNumber` => `DemoRoute.InputNumberLegacy` + tabulation. Nothing more.